cross/mpd/PLIST

```diff
@@ -1 +1,3 @@
 bin:bin/mpd
+rsc:share/doc/mpd/mpdconf.example
```

Comment:
If you want to include a configuration file I propose to create a working config file instead, if possible. We don't expect most users to use the command line, if there are options that need to be selected by a user you can ask the user at install time by using the wizard variables. You can use the transmission package as an example: https://github.com/SynoCommunity/spksrc/tree/master/spk/transmission/src/wizard